From e5ef172b8858b6e5ab1f0403b424f1ae7c4ee143 Mon Sep 17 00:00:00 2001
From: "Guoniu.Zhou" <guoniu.zhou@nxp.com>
Date: Fri, 12 Oct 2018 18:42:18 +0800
Subject: [PATCH 4862/5242] MLK-19915: m2m: correct V4L2_PIX_FMT_RGB32 to
 V4L2_PIX_FMT_XRGB32

commit  00070a131e833e59478c373ac8ac6b952cc08fbf from
https://source.codeaurora.org/external/imx/linux-imx.git

Because V4L2_PIX_FMT_RGB32 is deprecated and must not
be used by new drivers. V4L2_PIX_FMT_XRGB32 is used to
replace it.

Signed-off-by: Guoniu.Zhou <guoniu.zhou@nxp.com>
(cherry picked from commit e9e5c58dbea63bad8be43c924d35d499499b1804)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/media/platform/imx8/mxc-isi-m2m.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/imx8/mxc-isi-m2m.c b/drivers/media/platform/imx8/mxc-isi-m2m.c
index 25cd977..900a323 100644
--- a/drivers/media/platform/imx8/mxc-isi-m2m.c
+++ b/drivers/media/platform/imx8/mxc-isi-m2m.c
@@ -45,7 +45,7 @@ struct mxc_isi_fmt mxc_isi_input_formats[] = {
 	/* Pixel link input format */
 	{
 		.name		= "RGB32",
-		.fourcc		= V4L2_PIX_FMT_RGB32,
+		.fourcc		= V4L2_PIX_FMT_XRGB32,
 		.depth		= { 32 },
 		.color =	MXC_ISI_M2M_IN_FMT_XRGB8,
 		.memplanes	= 1,
-- 
1.7.9.5

